Doc/library/os.rst

```diff
@@ -4511,7 +4511,7 @@ written in Python, such as a mail server's external command delivery program.
       already more likely to experience deadlocks running such code.
 
       See `this discussion on fork being incompatible with threads
-      <https://discuss.python.org/t/33555>`_
+      <https://discuss.python.org/t/concerns-regarding-deprecation-of-fork-with-alive-threads/33555>`_
```

Comment:
fwiw the subject in the discuss URL changes as it is edited so it really ignored, looking for the /number regardless.  that redirect presents a present-time friendly URL that can be read but may not be canonical as editing the subject will make it redirect again in the future.  accepting regardless, subjects of old threads rarely change.

Reply:
Hmm it sounds as a good entry to ignore redirect via conf.py.